src/psyclone/dynamo0p3_builtins.py

```diff
@@ -198,6 +198,11 @@ def qr_required(self):
         ''' Built-ins do not currently require quadrature '''
         return False
 
+    @property
```

Comment:
The added code is all fine, but the years in the Copyright need to be updated to "2017-2020".

Reply:
Done.